tests/distributions/conftest.py

```diff
@@ -427,6 +427,23 @@ def __init__(self, loc, scale):
             scipy_arg_fn=lambda rate: ((np.array(rate),), {}),
             prec=0.08,
             is_discrete=True),
+    Fixture(pyro_dist=dist.Geometric,
+            scipy_dist=sp.geom,
+            examples=[
+                {'logits': [2.0],
```

Comment:
Can we also directly test with `probs=0.` and `probs=1.` or would that cause issues with scipy?

Reply:
I haven't tested but I guess it will return `-inf` in scipy and a negative value ~-20 in torch (which is similar to Bernoulli or other distributions with logits). Let me verify...

Reply:
@neerajprad Geometric will raise an error if `probs=0` in its construction. For `probs=1`, scipy returns -inf while Geometric returns a small negative number (in log scale). This is also the case of `torch.distributions.Bernoulli(probs=torch.tensor(0.)).log_prob(1.)`.

Reply:
Sounds good, thanks for checking. LGTM.